Contract Template/description.ext

```diff
@@ -30,3 +30,4 @@ enableTargetDebug = 1;
 // Other settings
 cba_settings_hasSettingsFile = 1;  // Enables use of "cba_settings.sqf", remove if file is not present (will crash)
```

Comment:
File can always be present but empty (probably good practice). I would separate this away from "user editable" section.